p2p/config.go

```diff
@@ -0,0 +1,30 @@
+package p2p
+
+var (
+	defaultListenAddress = "0.0.0.0"
+	defaultPort          = "0"
```

Comment:
1. I'm begging you :) , specify the default port, it can't be "0", ask @ayk if you don't know which one to use
2. Please use `int` because for all our configs we use int type for port. We must be consistent.

Reply:
used int as ip :)